WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -540,6 +542,15 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
 }
 
 - (UIView *)tableView:(UITableView *)tableView viewForFooterInSection:(NSInteger)section {
+    BlogDetailsSection *detailSection = self.tableSections[section];
+    NSString *footerTitle = detailSection.footerTitle;
+    if (footerTitle != nil) {
+        BlogDetailsSectionFooterView *footerView = (BlogDetailsSectionFooterView *)[tableView dequeueReusableHeaderFooterViewWithIdentifier:BlogDetailsSectionFooterIdentifier];
+        // If the next section has title, gives extra spacing between two sections.
+        BOOL doesNextSectionHaveTitle = (self.tableSections.count > section + 1) ? (self.tableSections[section + 1].title != nil): NO;
```

Comment:
What if we call this `BOOL` like the method argument? I like what you used in the row below: `shouldShowExtraSpacing`